src/main/features/eoi/views/typeEoi.njk

```diff
@@ -24,6 +24,8 @@
         {% include "includes/project_header.njk" %}
 
         <p class="govuk-body-l">{{ data.form.question }}</p>
+        <br>
```

Comment:
Don't use br tag, you have to use GDS <p> spacing. https://design-system.service.gov.uk/styles/spacing/